### PR TITLE
Update analytics configurations for version 3.0.0

### LINKED ISSUE #86: Create resources for API Manager 3.0.0
**Description:**
Ansible resources should be created for API Manager 3.0.0

### PR DESCRIPTION
**Pull request by Savidude**

### CODE REVIEW

File: roles/apim-analytics-dashboard/templates/carbon-home/conf/dashboard/deployment.yaml.j2 [hunk 1]
```diff
@@ -392,3 +358,41 @@ wso2.transport.http:
...
+
+## Authentication configuration
+auth.configs:
+  type: apim
+  ssoEnabled: true
+  properties:
+    adminScope: apim_analytics:admin_carbon.super
+    allScopes: apim_analytics:admin apim_analytics:product_manager apim_analytics:api_developer apim_analytics:app_developer apim_analytics:devops_engineer apim_analytics:analytics_viewer apim_analytics:everyone openid apim:api_view apim:subscribe
+    adminServiceBaseUrl: {{ admin_service_url }}
+    adminUsername: admin
```
Comment (chirangaalwis) [inline roles/apim-analytics-dashboard/templates/carbon-home/conf/dashboard/deployment.yaml.j2]: @Savidude shall we parameterize the username and password values as well?

File: roles/apim-analytics-dashboard/templates/carbon-home/conf/dashboard/deployment.yaml.j2 [hunk 2]
```diff
@@ -392,3 +358,41 @@ wso2.transport.http:
...
+    adminPassword: admin
+    kmDcrUrl: {{ admin_service_url }}/client-registration/v0.15/register
+    kmTokenUrlForRedirection: https://localhost:9443/oauth2
```
Comment (chirangaalwis) [inline roles/apim-analytics-dashboard/templates/carbon-home/conf/dashboard/deployment.yaml.j2]: @Savidude shall we parameterize this to allow providing the Admin service URL?

File: roles/apim-analytics-dashboard/templates/carbon-home/conf/dashboard/deployment.yaml.j2 (hunk 1 above)
Comment (Savidude) [reply roles/apim-analytics-dashboard/templates/carbon-home/conf/dashboard/deployment.yaml.j2]: Fixed https://github.com/wso2/ansible-apim/pull/92/commits/33592d18770430db5df84a8aabbb7aca1c6b2b3c

File: roles/apim-analytics-dashboard/templates/carbon-home/conf/dashboard/deployment.yaml.j2 (hunk 2 above)
Comment (Savidude) [reply roles/apim-analytics-dashboard/templates/carbon-home/conf/dashboard/deployment.yaml.j2]: Fixed https://github.com/wso2/ansible-apim/pull/92/commits/33592d18770430db5df84a8aabbb7aca1c6b2b3c

**Pull request merged**